h/static/scripts/group-forms/utils/api.ts

```diff
@@ -0,0 +1,89 @@
+/* A successful response from h's create-new-group API:
+ * https://h.readthedocs.io/en/latest/api-reference/v2/#tag/groups/paths/~1groups/post
+ */
+export type CreateGroupAPIResponse = {
```

Comment:
```suggestion
/**
 * A successful response from h's create-new-group API:
 * https://h.readthedocs.io/en/latest/api-reference/v2/#tag/groups/paths/~1groups/post
 */
export type CreateGroupAPIResponse = {
```

JSDoc-style comments are either a single line of the form `/** Short summary. */` or a multi-line comment of the form:

```
/**
 * Short summary.
 *
 * More details...
 */
```

Reply:
Done